types/src/traits.rs

```diff
@@ -50,8 +50,3 @@ pub trait SubscriptionClient: Client {
 	where
 		Notif: DeserializeOwned;
 }
-
-/// JSON-RPC server interface for managing method calls.
-pub trait RpcMethod<R, E>: Fn(RpcParams) -> Result<R, E> + Send + Sync + 'static {}
```

Comment:
I removed this because I had to use the explicit type anyway [here](https://github.com/paritytech/jsonrpsee/pull/350/files#diff-7cf339924bf9f55b7b2a32c481d69d8e70114e91a0c074b7228bf7f80f964f8aR59) because rustc could not infer the types anyway. Not sure how big of a deal this is though?